message_sender/models.py

```diff
@@ -55,7 +55,7 @@ class Inbound(models.Model):
     transport_type = models.CharField(null=True, blank=True, max_length=200)
     helper_metadata = JSONField()
     updated_at = models.DateTimeField(auto_now=True)
-    created_at = models.DateTimeField(auto_now_add=True)
+    created_at = models.DateTimeField(auto_now_add=True, db_index=True)
```

Comment:
I know HelloMama has very few Inbounds but are we sure adding these indexes to MomConnect won't be problematic?